src/index.ts

```diff
@@ -95,65 +95,81 @@ export class WalletPluginWebAuthenticator extends AbstractWalletPlugin implement
         ui?: UserInterface
     ): Promise<{payload: CallbackPayload}> {
         return new Promise((resolve, reject) => {
-            try {
-                // Show status message using WharfKit UI
-                ui?.status('Opening authenticator popup...')
+            const t = ui?.getTranslate(this.id)
 
-                const t = ui?.getTranslate(this.id)
+            // Show status message using WharfKit UI
+            ui?.status('Opening wallet window...')
```

Comment:
Was just thinking that maybe we could call it the wallet window, lmk what you guys think.